src/api/app/helpers/webui/user_helper.rb

```diff
@@ -67,7 +67,11 @@ def user_with_realname_and_icon(user, opts = {})
         printed_name = "#{realname} (#{user.login})"
       end
 
-      user_image_tag(user) + ' ' + link_to(printed_name, user_show_path(user))
+      if opts[:no_icon]
+        link_to(printed_name, user_show_path(user))
+      else
+        user_image_tag(user) + ' ' + link_to(printed_name, user_show_path(user))
```

Comment:
I would interpolate this: `"#{user_image_tag(user)} #{link_to(printed_name, user_show_path(user))}"`